lib/compat.php

```diff
@@ -228,6 +228,25 @@ function gutenberg_get_post_type_capabilities( $user, $name, $request ) {
 	return $value;
 }
 
+/**
+ * Make sure oEmbed REST Requests apply the WP Embed security mechanism for WordPress embeds.
+ *
+ * TODO: This is a temporary solution. This code should be included in WordPress Core.
+ *
+ * @since ?
```

Comment:
While you're here you can go ahead and set this to the next release version number (2.1.0).